app/assets/stylesheets/user.scss

```diff
@@ -58,3 +29,34 @@
   text-align: center;
 }
 
+.big-error {
+  color: $google-red;
+  font-size: 1em;
+}
+
+.small-error {
+  @extend .pull-right;
+  color: $google-red;
+  font-size: 0.8em;
+}
+
+.forgot-password {
+  #show-password:checked + [for="show-password"]::before {
+    border-right: 2px solid $my-black;
```

Comment:
Properties should be ordered border-bottom, border-right, -moz-transform, -ms-transform, -o-transform, -webkit-transform